src/components/Rockets/Rockets.js

```diff
@@ -0,0 +1,25 @@
+import './Rockets.css';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets } from '../../redux/Rockets/rocketsSlice';
+import Rocket from './Rocket';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rocketsData = useSelector((state) => state.rockets);
+  useEffect(() => {
+    if (!rocketsData.length) {
+      dispatch(fetchRockets());
+    }
+  });
```

Comment:
- [OPTIONAL] Same thing here, I think it would be better to add `rocketsData` and `dispatch` to the dependency array of useEffect because they are used inside the useEffect Hook 👍

```
  useEffect(() => {
    if (!rocketsData.length) {
      dispatch(fetchRockets());
    }
  }, [rocketsData, dispatch]);
```